.github/workflows/e2e.yml

```diff
@@ -68,6 +69,9 @@ jobs:
           COMMIT_INFO_MESSAGE: ${{ github.event.pull_request.title }}
 
           # Visual Testing config
-          PERCY_TOKEN: ${{ secrets.PERCY_TOKEN }}
-          PERCY_PARALLEL_TOTAL: 3 # remember to keep this synced with matrix.containers
-          PERCY_BRANCH: ${{ github.head_ref }}
+          HAPPO_API_KEY: ${{ secrets.HAPPO_API_KEY }}
+          HAPPO_API_SECRET: ${{ secrets.HAPPO_API_SECRET }}
+
+      - name: Finalize Happo
```

Comment:
Unrelated to the failure, but this step needs the HAPPO_API_SECRET and HAPPO_API_KEY env variables as well.

Reply:
Of course 🤦‍♂️ . They are now shared across the whole job properly. Thanks!